### PR TITLE
Newsletter for 2019-12-03.md

### PR DESCRIPTION
**Pull request by Nyco**
Initial commit, I'll add the pictures

### CODE REVIEW

File: content/posts/newsletter/2019-12-03.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro 
+
+Welcome to the XMPP newsletter covering the month of November 2019.
+
+Help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
```
Comment (guusdk) [inline content/posts/newsletter/2019-12-03.md]: "of which the process is fully documented"

File: content/posts/newsletter/2019-12-03.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Edivaldo Brito has written in Portugese "[Como instalar o moderno cliente Jabber/XMPP Dino no Linux via Snap](https://www.edivaldobrito.com.br/cliente-jabber-xmpp-dino-no-linux/)" ("How to install the modern Jabber/XMPP Dino client on Linux via Snap").
+
+Last month, the XMPP newsletter has been translated by community members:
+* French: [Lettre d’information XMPP, 8 novembre 2019, sprints, IoT et le début de Twitter](https://linuxfr.org/news/lettre-d-information-xmpp-8-novembre-2019-sprints-iot-et-le-debut-de-twitter)
+* German: [XMPP Newsletter, 08. November 2019, Sprints, IoT und die Anfänge von Twitter](https://www.jabber.de/xmpp-newsletter-08-november-2019-sprints-iot-und-die-anfaenge-von-twitter/)
+* Spanish: [XMPP Newsletter, 08 Nov 2019, Sprints, IoT, y los primeros Twitter](https://www.jabber.de/es-xmpp-newsletter-08-nov-2019-sprints-iot-y-los-primeros-twitter/)
+
+
+We encourage you all to contribute to this translation effort!
```
Comment (guusdk) [inline content/posts/newsletter/2019-12-03.md]: Maybe: "We are extremely grateful for these contributions! If you can contribute a translation in your own language, please contact <someone>"

File: content/posts/newsletter/2019-12-03.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+### Clients and applications 
+
+
+
+[Movim 0.16 – Cesco](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/e10a8998-5e4d-4b52-9b9e-2f103b446d94) has been released, with Drawing and content sharing, Attachments, Chat improvements, and Chats, chatrooms list improvements, and Search, plus more.
+
+[gajim.org](https://gajim.org) features a new website. It is planned to publish a post about Gajim's development every month, starting with [October](https://gajim.org/post/2019-11-06-development-news-october/) and [November](https://gajim.org/post/2019-11-25-development-news-november/).
+
+Goffi has published his [SàT progress note 2019-W48](https://www.goffi.org/b/R5PeQUPBCSZWvabRNXtAE9/progress-note).
```
Comment (guusdk) [inline content/posts/newsletter/2019-12-03.md]: Describe Goffi's role for people that don't know him.

File: content/posts/newsletter/2019-12-03.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Changelog:
+Accepted by vote of Council on 2019-10-23. (XEP Editor (jcb))
+
+URL: https://xmpp.org/extensions/xep-0424.html
+
+#### Message Moderation 
+
+
+
+rsion 0.1.0 of XEP-0425 (Message Moderation) has been released.
```
Comment (guusdk) [inline content/posts/newsletter/2019-12-03.md]: rsion? Version?

**Comment by licaon-kter** [comment]
...and remove the ejabberd mention since already done in Nov'19.

File: content/posts/newsletter/2019-12-03.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+## Intro 
+
+Welcome to the XMPP newsletter covering the month of November 2019.
+
+Help us sustain this as a community effort, of which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Edivaldo Brito has written in Portugese "[Como instalar o moderno cliente Jabber/XMPP Dino no Linux via Snap](https://www.edivaldobrito.com.br/cliente-jabber-xmpp-dino-no-linux/)" ("How to install the modern Jabber/XMPP Dino client on Linux via Snap").
```
Comment (Ppjet6) [inline content/posts/newsletter/2019-12-03.md]: Do we want to poke the author of the article to fix the url maybe? `no` / `on`

File: content/posts/newsletter/2019-12-03.md (hunk 2 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: "We are extremely grateful for these contributions! If you can contribute a translation in your own language, please [contact the CommTeam](https://xmpp.org/about/xsf/comm-team.html)."
?

File: content/posts/newsletter/2019-12-03.md (hunk 3 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: 👍

File: content/posts/newsletter/2019-12-03.md (hunk 4 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: Thx!
Corrected in "a nother" commit. 😄

File: content/posts/newsletter/2019-12-03.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+![Dino](https://framapic.org/OewIIui3hyiP/UiRW9IhRFoBF.png)
```
Comment (Ppjet6) [inline content/posts/newsletter/2019-12-03.md]: I'm not sure relying on external services to host pictures is a great idea. Plus Framasoft announced they were stopping their services at some point.

File: content/posts/newsletter/2019-12-03.md (hunk 5 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: Sure, please do.
Question: If the URL is fixed at a later time, the old/wrong URL should redirect to the right one... right?

File: content/posts/newsletter/2019-12-03.md (hunk 6 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: Right.
How do you manage pictures in GitHub and Pelican?

File: content/posts/newsletter/2019-12-03.md (hunk 5 above)
Comment (Ppjet6) [reply content/posts/newsletter/2019-12-03.md]: That depends on the author and how they implement the change.

Comment (Ppjet6) [reply content/posts/newsletter/2019-12-03.md]: gtranslate says that's how you say "no" in portuguese, sorry :P

File: content/posts/newsletter/2019-12-03.md (hunk 6 above)
Comment (Nyco) [reply content/posts/newsletter/2019-12-03.md]: OK to change, but quite complex.

**Pull request merged**